### DISCUSSION TITLE
Focal point in (split) background images

### DISCUSSION THREAD

**Discussion by sbibauw** [marp-team/marp#134] [Q&A] [answered]
Is there an easy way to specify a different focal point in a background image (set with `![bg](image.jpg)`), in particular a split background image (`![bg left](image.jpg)`)? By default, Marp crops the image and focuses on the center (which is the best choice 95% of the time), but sometimes I'd like for it to crop from the left/right, or top/bottom in case of vertically cropped images. Is there some kind of 

    ![bg left focus:left](image.jpg)

![Landscape-Color](https://user-images.githubusercontent.com/4708459/124334547-0bb84800-db5d-11eb-8feb-df004f28f278.jpg)

?

**Comment by yhatt** [comment, marked as answer]
We do not provide a keyword for setting the focal point, but you can override the style for advanced background image through `<style scoped>` for now.

```markdown
![bg left](image.jpg)

<style scoped>
figure {
  background-position: left top !important;
}
</style>
```

![](https://user-images.githubusercontent.com/3993388/124757114-3b55be80-df68-11eb-8d75-66d2c5f47045.gif)
